index.d.ts

```diff
@@ -10,4 +10,6 @@ import {Chai} from './lib/Chai';
 
 declare var chai: Chai;
 
-export = chai;
+export default chai;
```

Comment:
Sounds good. The other things is about the tslint. The format might be a bit off.